### PR TITLE
Feature/event details view

### PR DESCRIPTION
**Pull request by kevinhartmann23**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
Adds event details
Adds agenda to-do list

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
Needs some styling matches to regular screen and responsiveness

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/agenda/AgendaItem.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+
+function AgendaItem({info}) {
+  const {id, startTime, title, description, isCompleted} = info
+  const [completed, setCompleted] = useState(isCompleted)
+
+  useEffect(() => {
+    //INSERT USECONTEXT TO UPDATE SUBEVENT ITEM
+  }, [completed])
```
Comment (DrewBradley) [inline src/components/agenda/AgendaItem.js:9]: Nice! Great work!

File: src/components/agenda/AgendaSideBar.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    e.preventDefault()
+    const newAgendaItem = formatAgendaItem()
+    updateAgenda(newAgendaItem)
+    titleRef.current.value = ''
+    descriptionRef.current.value = ''
+    setTime('')
+  }
+
+  const convertTime = (time) => {
+    const splitTime = time.split(':')
```
Comment (DrewBradley) [inline src/components/agenda/AgendaSideBar.js:18]: This is a smart function! Great job!

File: src/components/contacts/Contacts.css [hunk 3]
```diff
@@ -44,7 +44,7 @@
   margin: 0;
 }
 
-a {
+.Contacts a {
```
Comment (DrewBradley) [inline src/components/contacts/Contacts.css:47]: THANK YOU!!!!

**Review by DrewBradley — APPROVED** [review]

File: src/components/App.js [hunk 4]
```diff
@@ -17,6 +19,8 @@ function App() {
         <Route path="/calendar" component={Calendar} />
         <Route path="/new-event" component={Form} />
         <Route path="/contacts" component={Contacts} />
+        <Route path="/event-details" component={Event} />
```
Comment (kristenmb) [inline src/components/App.js:22]: if we want we can probably adjust this to be a match parameter but this is also probably fine for now!

File: src/components/agenda/Agenda.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React, {useState, useEffect} from 'react';
+import AgendaList from './AgendaList'
+import AgendaSideBar from './AgendaSideBar'
+import './Agenda.css'
+
+function Agenda({setAgenda, agenda}) {
+
+  const updateAgenda = (listItem) => {
```
Comment (kristenmb) [inline src/components/agenda/Agenda.js:8]: I like how you've broken this all out - it makes it very easy to use/follow

File: src/components/agenda/AgendaSideBar.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    if (hours > 0 && hours <= 12) {
+      timeValue = "" + hours;
+    } else if (hours > 12) {
+      timeValue = "" + (hours - 12);
+    } else if (hours === 0) {
+      timeValue = "12";
+    }
+
+    timeValue += (minutes < 10) ? ":0" + minutes : ":" + minutes
+    timeValue += (hours >= 12) ? "PM" : "AM"
```
Comment (kristenmb) [inline src/components/agenda/AgendaSideBar.js:34]: This looks really great. I like how you've set this up. If we have time down the line might be worth splitting some of these into a utils file.

**Review by kristenmb — APPROVED** [review]
These changes are great - its all coming together!

**Review by richardltyler — APPROVED** [review]
Looks great!

**Pull request merged**